src/components/reviews.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import { fetchReviews } from "../api";
+
+const Reviews = () => {
+    const [reviewsData, setReviews] = useState([]);
+
+    useEffect(() => {
+        fetchReviews().then((data) => {
+            setReviews(data);
+        });
+    });
+
+    return (
+        <div>
```

Comment:
Please use conditional rendering here so the user will be notified if the reviews are still loading.